### ISSUE TITLE
fluid.ampslice~/fluid.ampgate~ default parameters

### ISSUE THREAD

**Issue by rconstanzo** [flucoma/flucoma-core#2]
As per this thread on the forum:
https://discourse.flucoma.org/t/fluid-ampslice-fluid-ampgate-default-parameters/475

The default parameters for `fluid.ampslice~` and `fluid.ampgate~` are not in line with the other Max objects in they have default parameters that *never* produce results with any incoming signal. They should at minimum be in the ballpark of useful figures, potentially from the "musical example" tabs from each helpfile.

**Comment by weefuzzy** [comment]
Transferred as this is not specific to Max.

**Comment by tremblap** [comment]
I would mark as invalid, since there is no way to agree what default should be.

**Comment by rconstanzo** [comment]
I think we can all agree that they shouldn't be what they are now!

I mean, why don't fft values default to 1 1 1, and why cap the `onthreshold` to 144, why not 1000?

At minimum, it should be like other objects, where there exists a signal, in the universe, that can return a result. For neither of these is that the case.

**Comment by tremblap** [comment]
> I think we can all agree that they shouldn't be what they are now!

nope. The current default is what I think is the best. Good luck trying to convince me otherwise, but by saying it is obvious won't cut it :)

**Comment by rconstanzo** [comment]
How do you decide what the defaults for each object are? Is it random? Because that is currently the situation.

Some objects, most objects in fact, have defaults that produce results for a fair amount of material. These two do not.

![Screenshot 2020-05-16 at 2 05 34 pm](https://user-images.githubusercontent.com/2747015/82120493-6c7c9980-977e-11ea-9188-018465555939.png)

Nothing can satisfy this criteria. Like literally.

Why is this object different from every other object in the toolbox?

**Comment by tremblap** [comment]
> Is it random? Because that is currently the situation.

I get from this 'exchange' that you are (rudely) saying you don't understand the decisions. Good to know in all cases.

**Comment by rconstanzo** [comment]
Not trying to be rude, just pointing out that these default settings are very different from everything else in the toolbox.

This is also the "newest" of the objects, so it raises some concerns that the default values going forward will be along similar lines.

**Comment by weefuzzy** [comment]
Right, I'm calling timeout here. I think we've already agreed in the linked thread to talk this over between ourselves, and the conversation here immediately went in a less productive direction. 

I can see the general point but, fair warning, it might still be a won't-fix.

**Comment by rconstanzo** [comment]
Yeah my bad. I just moved it onto github since this is the place for issues now.

**Comment by tremblap** [comment]
my bad too. grumpy day it seems. sorry all.

**Comment by weefuzzy** [comment]
Big ❤️ all round. It's useful to have it as a feature request here.

**Comment by rconstanzo** [comment]
I hadn't checked the default params of `fluid.ampslice~` in ages, but when making a new patch to test with the other day I realized it's still the same as my screenshot above.

It would be great if it had some default params that could do _something_ on any bit of audio. (mainly so when I instantiate a new one, I can use those points to start messing with, rather than having go figure out the domain of numbers necessary)

**Comment by rconstanzo** [comment]
I volunteer the following as an ok set of "generic" values.

1fluid.ampslice~ @fastrampup 10 @fastrampdown 1000 @slowrampup 4410 @slowrampdown 4410 @floor -70 @onthreshold 15 @offthreshold 10 @highpassfreq 801
